dev-debug/bpftrace/metadata.xml

```diff
@@ -4,10 +4,15 @@
   <maintainer type="person">
     <email>chutzpah@gentoo.org</email>
   </maintainer>
+  <maintainer type="person" proxied="yes">
+    <email>holger@applied-asynchrony.com</email>
+    <name>Holger Hoffstätte</name>
+  </maintainer>
```

Comment:
I'd suggest you putting yourself as top maintainer, that will get yourself assigned at b.g.o, got more privilege to handle the bug (e.g: edit permission)..

also you need to add 'proxy-maint' info into metadata.xml..

I'm ping-ing @chutz to see if he is ok with me to add myself to  metadata, so I can keep my eyes on this package and help you do proxy commit in the future..

Reply:
Done, thank you! I didn't know the order of entries has implications (typically it does not in XML). Thanks for taking a look.